packages/core/src/node/dev.ts

```diff
@@ -18,14 +18,16 @@ interface DevOptions {
 
 export async function dev(options: DevOptions): Promise<ServerInstance> {
   const { docDirectory, config, extraBuilderConfig } = options;
-  const base = config?.base ?? '';
   const isProd = false;
   const pluginDriver = new PluginDriver(config, isProd);
   await pluginDriver.init();
 
   try {
     const modifiedConfig = await pluginDriver.modifyConfig();
     await pluginDriver.beforeBuild();
+
+    rsbuildLogger.override(logger);
```

Comment:
Both Rsbuild and Rspress uses `rslog`, I think the override is no longer needed.